scripts/import_election_program.py

```diff
@@ -0,0 +1,224 @@
+            cur_chapter["is_parent"] = True
+            cur_chapter["number"] += ".0"
+
+        results += sub_chapters
+
+    return results
+
+
+@log_call
+def load_election_program(filepath, layer_depth):
+    with open(filepath) as program_file:
+        with start_action(log_level="INFO", action_type="load_file"):
+            content = program_file.read()
+
+    # Compile regex patterns
+    regex_list = prepare_regex()
+
+    # Start recursive parsing of layers
+    return parse_layer(content, 1, regex_list, [], layer_depth)
+
+
+@log_call
+def insert_proposition(subject_area, proposition_type, voting_phase, proposition, identifier, data_tag_regex):
+
+    # Remove data-section tags
+    text = data_tag_regex.sub("", proposition["content"])
+
+    if text is None or len(text) == 0 or text.isspace():
+        return False
+
+    # Make title
+    title = "Streichung: " + proposition["number"] + " " + proposition["name"]
+
+    chapters = ""
+    for parent in proposition["parent_chapters"]:
+        chapters += "**" + parent["number"] + "** " + parent["name"] + "\n"
+
+    chapters += "**" + proposition["number"] + " " + proposition["name"] + "**\n"
+
+    for child in proposition["sub_chapters"]:
+        chapters += "**" + child["number"] + "** " + child["name"] + "\n"
+
+    abstract = "Streichung von Kapitel " + proposition["number"] + " des Wahlprogramms zur Bundestagswahl 2021"
+
+    content = "Der Bundesparteitag möge beschließen, im Wahlprogramm zur Bundestagswahl\n"
+
+    if proposition["is_parent"]:
+        content += "in "
+
+    content += "Kapitel **" + proposition["number"] + " " + proposition["name"] + "** "
+
+    if proposition["is_parent"]:
+        content += "den Text vor den Unterabschnitten "
+
+    content += "zu streichen.\n\n"
+
+    motivation = "Der Bundesparteitag 2019.2 hat beschlossen, dass alle Teile des Programms zur Bundestagswahl 2017 " \
+                 "zur Streichung angemeldet werden um eine Überarbeitung des Programms zur Bundestagswahl 2021 sicherzustellen."
+
+    motivation += "\n#### Einordnung\n"
+    motivation += chapters
+
+    motivation += "\n#### Kapitelinhalt\n"
+
+    motivation += text
+
+    voting_identifier = "WP" + str(identifier).zfill(3)
+
+    ballot = Ballot(area=subject_area, name=voting_identifier, voting_type=VotingType.ASSEMBLY,
+                    voting=voting_phase, proposition_type=proposition_type)
+
+    proposition_obj = Proposition(title=title, abstract=abstract, content=content, motivation=motivation,
+                                  voting_identifier=voting_identifier, ballot=ballot, status=PropositionStatus.SCHEDULED)
+
+    # Add tags, some defaults + chapter name + parent chapter names (tags can only have a maximum length of 64 characters)
+    tags = ["wahlprogrammantrag", "bundestagswahl", "streichung"]
+
+    # Only use top chapter
+    if len(proposition["parent_chapters"]) > 0:
+        chapter_tag = proposition["parent_chapters"][0]
```

Comment:
chapter_tag is a dict here, should be:

```suggestion
        chapter_tag = proposition["parent_chapters"][0]["name"]
```